torchbearer/callbacks/tensor_board.py

```diff
@@ -164,6 +164,20 @@ def on_start(self, state):
         self.log_dir = os.path.join(self.log_dir, state[torchbearer.MODEL].__class__.__name__ + '_' + self.comment)
         self.writer = self.get_writer(visdom=self.visdom, visdom_params=self.visdom_params)
 
+    @staticmethod
+    def add_metric(add_fn, tag, metric, *args, **kwargs):
```

Comment:
In principle this could apply to other things than metrics, like lists or dicts of images etc. I've put it in AbstractTensorboard for this reason.